app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupUnlockScreen.kt

```diff
@@ -0,0 +1,273 @@
+            isChecked = state.isUnlockWithBiometricsEnabled,
+            onDisableBiometrics = handler.onDisableBiometrics,
+            onEnableBiometrics = handler.onEnableBiometrics,
+            modifier = Modifier
+                .testTag(tag = "UnlockWithBiometricsSwitch")
+                .fillMaxWidth()
+                .padding(horizontal = marginHorizontal),
+        )
+        BitwardenUnlockWithPinSwitch(
+            isUnlockWithPasswordEnabled = state.isUnlockWithPasswordEnabled,
+            isUnlockWithPinEnabled = state.isUnlockWithPinEnabled,
+            onUnlockWithPinToggleAction = handler.onUnlockWithPinToggle,
+            modifier = Modifier
+                .testTag(tag = "UnlockWithPinSwitch")
+                .fillMaxWidth()
+                .padding(horizontal = marginHorizontal),
+        )
+
+        Spacer(modifier = Modifier.height(height = 24.dp))
+        BitwardenFilledButton(
+            label = stringResource(id = R.string.continue_text),
+            onClick = handler.onContinueClick,
+            isEnabled = state.isContinueButtonEnabled,
+            modifier = Modifier
+                .testTag(tag = "ContinueButton")
+                .fillMaxWidth()
+                .padding(horizontal = marginHorizontal),
+        )
+
+        Spacer(modifier = Modifier.height(height = 12.dp))
+        SetUpLaterButton(
+            onConfirmClick = handler.onSetUpLaterClick,
+            modifier = Modifier
+                .testTag(tag = "SetUpLaterButton")
+                .fillMaxWidth()
+                .padding(horizontal = marginHorizontal),
+        )
+
+        Spacer(modifier = Modifier.height(height = 12.dp))
+        Spacer(modifier = Modifier.navigationBarsPadding())
+    }
+}
+
+@Composable
+private fun SetUpLaterButton(
+    modifier: Modifier,
+    onConfirmClick: () -> Unit,
+) {
+    var displayConfirmation by rememberSaveable { mutableStateOf(value = false) }
+    if (displayConfirmation) {
+        @Suppress("MaxLineLength")
+        BitwardenTwoButtonDialog(
+            title = stringResource(id = R.string.set_up_unlock_later),
+            message = stringResource(
+                id = R.string.you_can_return_to_complete_this_step_anytime_from_account_security_in_settings,
+            ),
+            confirmButtonText = stringResource(id = R.string.confirm),
+            dismissButtonText = stringResource(id = R.string.cancel),
+            onConfirmClick = {
+                onConfirmClick()
+                displayConfirmation = false
+            },
+            onDismissClick = { displayConfirmation = false },
+            onDismissRequest = { displayConfirmation = false },
+        )
+    }
+
+    BitwardenTextButton(
+        label = stringResource(id = R.string.set_up_later),
+        onClick = { displayConfirmation = true },
+        modifier = modifier,
+    )
+}
+
+@Composable
+private fun ColumnScope.SetupUnlockHeaderPortrait() {
```

Comment:
This one is a little odd, normally we would require a `Modifier` to be passed in here but since this is an extension of `ColumnScope`, I skipped it. It really wouldn't make since without a top level component.